src/main/java/com/thebois/models/beings/AbstractBeing.java

```diff
@@ -90,6 +89,8 @@ public void setRole(final AbstractRole role) {
 
     @Override
     public void update() {
+        // Registers everytime because we serialize.
```

Comment:
A bit unclear comment, you could clarify it to "Registers every time because it might be registered to an old instance of the event bus (caused by saving/loading)."